app/models/submission.rb

```diff
@@ -0,0 +1,54 @@
+require 'securerandom'
+
+class Submission < ActiveRecord::Base
+  belongs_to :project
+  has_many :jobs, dependent: :destroy
+
+  store :job_attrs
+
+  attr_accessor :start_frame, :end_frame, :project_dir
+
+  def submit
+    parse_frames
+
+    job = new_job
+    success = job.submit(templated_content)
+
+    success
+  end
+
+  def never_submitted_status
+    'not submitted'
```

Comment:
If you are persisting 'not submitted' as a status, it might be preferable to instead subclass `OodCore::Job::Status` to add an additional state 'not_submitted'. But that decision would need to be made in light of how status management is being done across the app, which is something we can discuss.